SpatialGDK/Source/SpatialGDKFunctionalTests/SpatialGDK/SpatialTestTearOff/SpatialTestTearOff.cpp

```diff
@@ -0,0 +1,186 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialTestTearOff.h"
+
+#include "ReplicatedTearOffActor.h"
+#include "SpatialGDKFunctionalTests/SpatialGDK/TestActors/ReplicatedTestActorBase.h"
+
+#include "Kismet/GameplayStatics.h"
+
+/**
+ * This test automates the QA flow that checks if Actors can be prevented from replicating with bTearOff.
+ * NOTE: Due to adding wait steps before checking certain parts of the test, this test runs slow, in roughly 30 seconds. This can be improved by modifying the WorkerWaitForTimeStepDefinition
+ *       to wait for a smaller amount of time. 
+ *
+ * The test includes 1 Server and 2 Client workers.
+ * The flow is as follows:
+ * - Setup:
+ *  - A ReplicatedTearOffActor is placed in the map that is running the test.
```

Comment:
This makes it sound like the test does this, but it actually requires a map with it preplaced.
On a related note, any reason it needs to be a start-up actor? Is that the QA flow?

Reply:
I see it has both flows, so I would just reword this to say one needs to be preplaced in the map.

Reply:
Okay this is something I should have been specific about in the PR, there is currently no 'accepted' test flow for covering  the bTearOff QA workflow, this PR is basically trying to achieve that with an automated test. So yeah I thought that the 3 main flows would be: a startup Actor that calls TearOff on BeginPlay(), a dynamically spawned Actor with the same behaviour, and a dynamically spawned 'normal'replicated Actor that has TearOff called at some point. I have just updated the PR comment to make this more clear.

Reply:
Related to the wording, I agree it is confusing, I will update that, thank you for the suggestion!